Sources/ComposableArchitecture/Effects/Deferring.swift

```diff
@@ -0,0 +1,29 @@
+import Combine
+
+extension Effect {
+  /// Returns an effect that will be executed after given `dueTime`.
+  ///
+  ///     case let .textChanged(text):
+  ///       return environment.search(text)
+  ///         .map(Action.searchResponse)
+  ///         .deferred(for: 0.5, scheduler: environment.mainQueue)
```

Comment:
```suggestion
  /// ```swift
  /// case let .textChanged(text):
  ///   return environment.search(text)
  ///     .map(Action.searchResponse)
  ///     .deferred(for: 0.5, scheduler: environment.mainQueue)
  /// ```
```